pepdbagent/modules/project.py

```diff
@@ -228,14 +228,14 @@ def create(
 
         namespace = namespace.lower()
         if name:
-            name = name.lower()
-            proj_name = name
-            proj_dict[CONFIG_KEY]["name"] = proj_name
+            proj_name = name.lower()
         elif proj_dict[CONFIG_KEY]["name"]:
             proj_name = proj_dict[CONFIG_KEY]["name"].lower()
         else:
             raise ValueError(f"Name of the project wasn't provided. Project will not be uploaded.")
 
+        proj_dict[CONFIG_KEY]["name"] = proj_name
```

Comment:
Is there a similar default key for `"name"` ?

Reply:
done